### PR TITLE
spec: update instance tags

### PR DESCRIPTION
**Pull request by nicolasparada**
Update the OpenAPI spec to add support for updating tags on a core instance.

### CODE REVIEW

File: client/aggregator_test.go [hunk 1]
```diff
@@ -231,6 +231,7 @@ func TestClient_UpdateAggregator(t *testing.T) {
 
 	err = withToken.UpdateAggregator(ctx, created.ID, types.UpdateAggregator{
 		Name: ptrStr("test-aggregator-updated"),
+		Tags: ptrSliceStr([]string{"updatedtag"}),
```
Comment (niedbalski) [inline client/aggregator_test.go:234]: Please add some extra test with empty, nil, " ", "\t " and multiple tags.

**Review by niedbalski — CHANGES_REQUESTED** [review]
Missing a few test cases.

File: client/aggregator_test.go (hunk 1 above)
Comment (nicolasparada) [reply client/aggregator_test.go]: Is it really needed that much validation here?

Actual testing happens on the server.
These are the client tests, we just need to know that the endpoint is connected.

I updated the test to refetch the core instance after the update and check that it was indeed updated.

**Review by niedbalski — APPROVED** [review]

File: client/aggregator_test.go (hunk 1 above)
Comment (nicolasparada) [reply client/aggregator_test.go]: I'm totally ok moving the tests here btw 
@niedbalski 
It would be a lot less of code repetition.
But the flow of doing two PRs on different repos prevent it :(
